composer.json

```diff
@@ -37,6 +37,7 @@
   "require": {
     "php": ">=7.2.0",
     "ext-json": "*",
+    "ext-mbstring": "*",
```

Comment:
As far as I know always presented in PHP, at least in the newer versions (and also a discussion in PHP group to make json presented as well in PHP 8.0)

Reply:
ok this can be removed if you say so